tough/src/lib.rs

```diff
@@ -874,10 +889,105 @@ fn load_targets<T: Transport>(
 
     // Now that everything seems okay, write the targets file to the datastore.
     datastore.create("targets.json", &targets)?;
```

Comment:
Move these two lines above the `// 4.5` comment, and remove the now-incorrect `does not yet handle delegated roles` comment.